Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml

```
@@ -566,6 +566,31 @@
         - ReagentId: Amatoxin
           Quantity: 6
 
+- type: entity
+  name: scrambled eggs
+  parent: FoodBowlBase
+  id: FoodMealScrambledEggs
+  description: Some cooked eggs with salt. Yummy!
+  components:
+  - type: FlavorProfile
+    flavors:
+      - egg
+      - salty
+  - type: Sprite
+    layers:
+    - state: bowl
+    - state: scrambled-eggs
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 10
```

```suggestion
        maxVol: 10
```
This is case-sensitive and causes errors.